### PR TITLE
Final cleanup

### PR DESCRIPTION
**Pull request by blurpesec**
### Description
Change static text to translations in defi zap flow. Enables defi zap in beta.

### TODOs Before Merge:
- [x] Local tx history
- [x] Auto-scan tokens
- [x] Figure out About Zap Panel illustrations.
- [x] Get design and copy reviews.

### CODE REVIEW

File: common/v2/features/DeFiZap/components/ZapCard.tsx [hunk 1]
```diff
@@ -53,14 +53,26 @@ const ZapCardContent = styled('div')`
   flex-direction: column;
 `;
 
-const ZapCardContentText = styled.div`
-  padding: ${SPACING.SM} 0px;
+const ZapCardContentText = styled.p`
```
Comment (FrederikBolding) [inline common/v2/features/DeFiZap/components/ZapCard.tsx]: Is there a reason we don't use `Typography` for all of these texts? (in general throughout the flow)

File: common/v2/features/DeFiZap/components/ZapSelection.tsx [hunk 2]
```diff
@@ -59,12 +57,12 @@ const BreakdownImg = styled.img`
...
 `;
 
-const DetailsSection = styled(SSection)`
-  display: flex;
-  flex-direction: row;
-  align-items: flex-start;
-  justify-content: space-between;
-`;
+// const DetailsSection = styled(SSection)`
+//   display: flex;
```
Comment (FrederikBolding) [inline common/v2/features/DeFiZap/components/ZapSelection.tsx]: Remove if unused

File: common/v2/features/DeFiZap/config.ts [hunk 3]
```diff
@@ -61,118 +65,115 @@ export const fetchZapRiskObject = (riskLevel: number) => {
...
+    positionDetails: () =>
+      BearishIndicator({ text: translateRaw('ZAP_POSITION_BEARISH', { $asset: 'ETH' }) })
   }
 };
 
 export const riskAndReward: RiskAndReward[] = [
   {
     text:
-      '**Smart Contract:** There is a chance that  smart contracts get hacked and you lose all your money',
+      '**Smart Contract:** There is a chance that smart contracts get hacked and you lose all your money.',
```
Comment (FrederikBolding) [inline common/v2/features/DeFiZap/config.ts]: Can we translate these too?

Comment (FrederikBolding) [reply common/v2/features/DeFiZap/config.ts]: Same for other strings in this file.

File: common/v2/features/SendAssets/stateFactory.tsx [hunk 4]
```diff
@@ -110,7 +111,14 @@ const TxConfigFactory: TUseStateReducerFactory<State> = ({ state, setState }) =>
...
           state.txConfig.senderAccount,
-          { ...txReceipt, stage: ITxStatus.PENDING } || {}
+          {
+            ...txReceipt,
+            to: state.txConfig.receiverAddress,
+            from: state.txConfig.senderAccount.address,
+            amount: state.txConfig.amount,
+            txType: ITxType.STANDARD,
+            stage: ITxStatus.PENDING
+          } || {}
```
Comment (FrederikBolding) [inline common/v2/features/SendAssets/stateFactory.tsx]: Why are we using OR here?

File: common/v2/features/DeFiZap/components/DashboardZapCTA.tsx [hunk 5]
```diff
@@ -1,57 +1,33 @@
...
-  };
-
   return (
     <DashboardPanel
-      heading={'Put your ETH to work for you'}
+      heading={translateRaw('ZAP_DASHBOARD_PANEL_HEADER')}
       headingRight={<DeFiZapLogo />}
       className={className}
-      footer={<CTAFooter />}
+      footer={<></>}
```
Comment (GuillaumeRx) [inline common/v2/features/DeFiZap/components/DashboardZapCTA.tsx]: Is this here for a good reason ?

File: common/v2/features/DeFiZap/components/ZapCard.tsx (hunk 1 above)
Comment (blurpesec) [reply common/v2/features/DeFiZap/components/ZapCard.tsx]: Afaict, we've not really stuck to using it. Didn't think it was that important of an addition :woman_shrugging:

File: common/v2/components/Tooltip.tsx [hunk 6]
```diff
@@ -3,14 +3,33 @@ import React from 'react';
 import { Tooltip as UITooltip } from '@mycrypto/ui';
 
 import questionSVG from 'assets/images/icn-question.svg';
+import informationalSVG from 'assets/images/icn-info-blue.svg';
+
+export enum IconID {
+  question = 'question',
```
Comment (FrederikBolding) [inline common/v2/components/Tooltip.tsx]: Do they need to be strings?

File: common/v2/features/DeFiZap/components/ZapIndicators.tsx [hunk 7]
```diff
@@ -15,19 +14,20 @@ interface SProps {
...
 
 const SImage = styled.img`
-  height: 24px;
-  width: 24px;
+  height: 16px;
+  width: 16px;
 `;
 
-const SText = styled(Typography)`
+const SText = styled.div`
```
Comment (FrederikBolding) [inline common/v2/features/DeFiZap/components/ZapIndicators.tsx]: You can do `Typography` with attribute `as=div` if needed.

File: common/v2/translations/lang/en.json [hunk 8]
```diff
@@ -653,6 +653,56 @@
...
+    "ZAP_DAI_UNIPOOL_NAME": "DAI Unipool",
+    "ZAP_DAI_UNIPOOL_DESCRIPTION": "Hedge against ETH price volatility and earn a variable interest rate (generally 5-25%) by lending on the Uniswap exchange protocol.",
+    "ZAP_DAI_UNIPOOL_BULLETPOINT_ONE": "Reduce your exposure to changes in the price of ETH.",
+    "ZAP_DAI_UNIPOOL_BULLETPOINT_TWO": "Earn interest by lending to the DAI pool on the Uniswap exchange protocol.",
+    "ZAP_COMPOUND_DAI_POOL_TITLE": "Interest-earning with DAI",
+    "ZAP_COMPOUND_DAI_POOL_NAME": "Compound DAI Pool",
+    "ZAP_COMPOUND_DAI_POOL_DESCRIPTION": "Earn interest of up to 8% a year by contributing to the Compound money market protocol. Remove your ETH price exposure.",
+    "ZAP_COMPOUND_DAI_POOL_BULLETPOINT_ONE": "Remove your exposure to changes in the price of ETH.",
+    "ZAP_COMPOUND_DAI_POOL_BULLETPOINT_TWO": "Contribute your DAI to a money market protocol to earn interest of up to 8% a year.",
+		"DEFI_RESERVE_MAP_ERR": "DeFiReserveMapService has already been instantiated.",
```
Comment (FrederikBolding) [inline common/v2/translations/lang/en.json]: Is the indentation on purpose?

File: common/v2/components/Tooltip.tsx (hunk 6 above)
Comment (blurpesec) [reply common/v2/components/Tooltip.tsx]: No, but the strings are easier to read imo.

File: common/v2/features/DeFiZap/components/ZapIndicators.tsx (hunk 7 above)
Comment (blurpesec) [reply common/v2/features/DeFiZap/components/ZapIndicators.tsx]: This one was part of Guillaume's styling fixes, so I'm not going to touch it on the off chance it triggers another design review :3

**Pull request merged**